tern/analyze/common.py

```diff
@@ -69,7 +69,7 @@ def load_packages_from_cache(layer):
     raw_pkg_list = cache.get_packages(layer.fs_hash)
     if raw_pkg_list:
         logger.debug(
-            'Loading packages from cache: layer \"%s\"', layer.fs_hash[:10])
+            'Loading packages from cache: layer \"%s\"', layer.layer_index)
```

Comment:
I would prefer not to have this changed because the cache is keyed based on the layer's `fs_hash`

Reply:
Fixed!